AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/AwsCryptographyMaterialProvidersOperations.dfy

```diff
@@ -277,13 +291,22 @@ module AwsCryptographyMaterialProvidersOperations refines AbstractAwsCryptograph
         Types.AwsCryptographicMaterialProvidersException(
           message := "Must initialize keyring with either branchKeyId or BranchKeyIdSupplier."));
 
+    var gracePeriod := GetIntFromOpt(input.gracePeriod, 10);
+    var graceInterval := GetIntFromOpt(input.graceInterval, 1);
+    var fanOut := GetIntFromOpt(input.fanOut, 20);
+    var inFlightTTL := GetIntFromOpt(input.inFlightTTL, 20);
```

Comment:
I moved it into the constructor, but we still have the defaults in two places, because theres no clean way to transition from foo(x : Option<T>) to bar(x : T := 42)